lib/hanami/generators/app/templates/application.html.slim.tt

```diff
@@ -0,0 +1,8 @@
+doctype html
+html
+  head
+    title
+      | <%= config[:classified_app_name] %>
+    - == favicon
```

Comment:
@davydovanton This should be changed into:

```slim
= favicon
```

With the actual line I got a Slim syntax error:

![screen shot 2016-02-09 at 11 28 55](https://cloud.githubusercontent.com/assets/5089/12913651/583ff140-cf20-11e5-8b60-1ab894030131.png)